### PR TITLE
Add lvm2

### PR DESCRIPTION
**Pull request by AkitoTakada**
I have confirmed that the file exists, but I have not tested it.
The reasons for not being able to test are as follows.
There is no logical volume in the emulator.
Also a logical volume cannot be created because there is no physical volume.
Therefore, it cannot be tested.

All of libaio, thin-provisioning-tools, and lvm2 are in the same situation.

```
root@qemuarm64:~# find / -name "*lvm*" 
/usr/lib/libdevmapper-event-lvm2.so.2.03
/usr/lib/device-mapper/libdevmapper-event-lvm2mirror.so
/usr/lib/device-mapper/libdevmapper-event-lvm2snapshot.so
/usr/lib/device-mapper/libdevmapper-event-lvm2vdo.so
/usr/lib/device-mapper/libdevmapper-event-lvm2thin.so
/usr/lib/device-mapper/libdevmapper-event-lvm2raid.so
/usr/lib/liblvm2cmd.so.2.03
/usr/sbin/lvmsadc
/usr/sbin/lvmdiskscan
/usr/sbin/lvmconfig
/usr/sbin/lvm
/usr/sbin/lvmsar
/usr/sbin/lvmdump
/lib/udev/rules.d/69-lvm-metad.rules
/lib/udev/rules.d/56-lvm.rules
/etc/lvm
/etc/lvm/profile/lvmdbusd.profile
/etc/lvm/lvm.conf
/etc/lvm/lvmlocal.conf
/etc/init.d/lvm2-monitor

root@qemuarm64:~# find / -name "*dmsetup*" 
/usr/sbin/dmsetup
```

### CODE REVIEW

File: recipes-debian/thin-provisioning-tools/thin-provisioning-tools_debian.bb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+DEBIAN_UNPACK_DIR = "${WORKDIR}/${BPN}-${PV}"
+S = "${WORKDIR}/${BPN}-${PV}"
+
+SRC_URI += " \
+           file://0001-fix-compile-failed-with-libc-musl.patch \
+           file://use-sh-on-path.patch \
+"
+
+UPSTREAM_CHECK_GITTAGREGEX = "v(?P<pver>\d+(\.\d+)+)"
```
Comment (masami256) [inline recipes-debian/thin-provisioning-tools/thin-provisioning-tools_debian.bb]: It can be removed

**Comment by masami256** [comment]
@AkitoTakada  This pr contains thin-provisioning-tools package which has its own pr(#138). If this pr's purpose is add lvm2 only, please remove thin-provisioning-tools recipes from this pr.

**Comment by AkitoTakada** [comment]
Reverted.
Should I create a new branch and request it?

**Comment by masami256** [comment]
> Reverted.
> Should I create a new branch and request it?

@AkitoTakada No, you don't have to.

**Comment by yuwata** [comment]
@masami256 In this case, we should use 'squash' merge.

**Comment by masami256** [comment]
this package depends on thin-provisioning-tools, so it should be merged after #138 has merged.

**Comment by masami256** [comment]
thanks!

**Pull request merged**